s2i/bin/run

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+#
+# S2I run script for the 's2i-haproxy-centos7' image.
+# The run script executes the server that runs your application.
+#
+# For more information see the documentation:
+#	https://github.com/openshift/source-to-image/blob/master/docs/builder_image.md
+#
+
+if [ -f /tmp/scripts/run ]; then
+  exec /tmp/scripts/run "$@"
+else
+  exec haproxy -f "$(pwd)/conf/haproxy.conf"
```

Comment:
I dont see this file provided by the test-app. How does it start then?

Reply:
Totally forgot about the test-app, going to add a minimal conf for basic testing.